Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -153,7 +155,13 @@ private void OnGameChange(GameRunLevelChangedEvent ev)
 
         message = SanitizeInGameOOCMessage(message);
 
-        switch (type)
+        var sendType = type;
+        // If we have OOC disabled for players, redirect dead/ghost LOOC to dead chat.
+        if (!_adminManager.IsAdmin(player) && !_configurationManager.GetCVar(CCVars.OocEnabled)
+            && (HasComp<GhostComponent>(source) || _mobStateSystem.IsDead(source)))
+            sendType = InGameOOCChatType.Dead;
```

Comment:
i feel like ghost LOOC should go to deadchat regardless of if OOC is disabled or not